targets/TARGET_Samsung/TARGET_SIDK_S5JS100/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if(${MBED_TOOLCHAIN} STREQUAL "ARM")
+    set(LINKER_FILE device/TOOLCHAIN_ARM_STD/sidk_s5js100.sct)
+    set(STARTUP_FILE device/TOOLCHAIN_ARM_STD/startup_sidk_s5js100.S)
+elseif(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
+    set(LINKER_FILE device/TOOLCHAIN_GCC_ARM/sidk_s5js100.ld)
+    set(STARTUP_FILE device/TOOLCHAIN_GCC_ARM/startup_sidk_s5js100.S)
+endif()
+
+set_property(GLOBAL PROPERTY MBED_TARGET_LINKER_FILE ${CMAKE_CURRENT_SOURCE_DIR}/${LINKER_FILE})
+
+target_include_directories(mbed-core
+    INTERFACE
+        .
+        device
+        modem
+        security_subsystem
+        security_subsystem/drivers
+)
+
+target_sources(mbed-core
+    INTERFACE
+# TODO CMAKE: The files commented below trigger a dependency on other libraries
+# they should be moved somewhere in the connectivity directory.
+        # dcxo_update.cpp
+        # mbed_main_init.cpp
+        # modem/modem_io_device.cpp
+        # modem/modem_link_device_shmem.cpp
+        # modem/s5js100_mbox_ipc.cpp
+        # modem/shmem_save.cpp
```

Comment:
@0xc0170 are we okay leaving a TODO comment in the configuration file?

Reply:
I've missed this. It would be the best to remove (I'll do it now) and create an issue for this to be fixed.

Reply:
Fixed now